general/server/plugins/index.md

```diff
@@ -105,6 +143,14 @@ Scrape poster images for movies, shows, and artists in your library from [fanart
 
 * [GitHub](https://github.com/jellyfin/jellyfin-plugin-fanart)
 
+#### IPTV
+
+[![Language](https://img.shields.io/github/languages/top/jellyfin/jellyfin-plugin-iptv.svg)](https://github.com/jellyfin/jellyfin-plugin-iptv)
+[![Contributors](https://img.shields.io/github/contributors/jellyfin/jellyfin-plugin-iptv.svg)](https://github.com/jellyfin/jellyfin-plugin-iptv)
+[![License](https://img.shields.io/github/license/jellyfin/jellyfin-plugin-iptv.svg)](https://github.com/jellyfin/jellyfin-plugin-iptv)
+
+Plugin to provide additional IPTV support.
```

Comment:
Hasn't this one been archived?

Reply:
i dunno,  never used it lol. I'll check and fix it

Reply:
I recently removed it because I thought that was the case, if it's still in use then I was wrong.

Reply:
i saw that it was in jelly-archive but yeah  afaik, it was in use. i never heard anything about it being deprecated. it was in 10.6.4's manifest.

Reply:
We talked about the deprecation on Matrix, apparently it's no longer required, but I didn't ask why.